src/components/date-picker/AccessiblePikaday.ts

```diff
@@ -0,0 +1,74 @@
+import Pikaday, { PikadayOptions } from 'pikaday';
+
+export interface AccessiblePikadayOptions extends PikadayOptions {
+    accessibleFieldEl?: HTMLElement | null | undefined;
+    datePickerButtonEl?: HTMLElement | null | undefined;
+}
+
+// An extended version of Pikaday to work when `isAccessible` prop is `true`
+class AccessiblePikaday extends Pikaday {
+    accessibleFieldEl: HTMLElement | null | undefined;
+
+    datePickerButtonEl: HTMLElement | null | undefined;
+
+    constructor(options: AccessiblePikadayOptions) {
+        super(options);
+        this.accessibleFieldEl = options.accessibleFieldEl;
+        this.datePickerButtonEl = options.datePickerButtonEl;
+
+        // Override behavior as if `options.field` and `options.bound` were set.
+        // See https://github.com/Pikaday/Pikaday/blob/master/pikaday.js#L671
+        //     https://github.com/Pikaday/Pikaday/blob/master/pikaday.js#L695-L703
+        if (this.accessibleFieldEl) {
+            document.body.appendChild(this.el);
+
+            this.accessibleFieldEl.addEventListener('blur', this.handleBlur);
+
+            this.hide();
+        }
+    }
+
+    handleBlur = () => {
+        // TODO: Test in IE11
```

Comment:
Did it work in IE11?

Reply:
It does work in IE11 but will never be called. This method is only used for a blur event listener to `this.accessibleFieldEl` which will not be set for IE11.